### PR TITLE
Param to list apps to preload into a wic image

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mike Sul <mike.sul@foundries.io>

### CODE REVIEW

File: subcommands/targets/image.go [hunk 1]
```diff
@@ -19,15 +23,28 @@ func init() {
 		Args:  cobra.ExactArgs(1),
 	}
 	cmd.AddCommand(imageCmd)
+	imageCmd.Flags().StringVarP(&appsShortlist, "apps", "", "",
+		"comma,separate,list of Target apps to preload into a resultant image")
```
Comment (doanac) [inline subcommands/targets/image.go:27]: I suppose we should say the default is "all apps"

**Comment by mike-sul** [comment]
> this is slightly orthogonal, but while we are touching the usage portion of this command. The word "target" (which i think is my fault) can be confusing. Maybe our usage statement should say `<target-name>` that's subtle but might help slightly. We could also include an example so that the user sees you need to use a full target name?

Maybe `<target-full-name>`?

> We could also include an example so that the user sees you need to use a full target name?
makes sense

File: subcommands/targets/image.go (hunk 1 above)
Comment (mike-sul) [reply subcommands/targets/image.go:27]: +

**Comment by mike-sul** [comment]
@doanac Updated, I think adding "Example" to the command is really great idea and we should do it for each command.

**Pull request merged**